Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -9098,7 +9092,6 @@
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_KEY_NSHumanReadableCopyright = "Copyright © 2023 Datadog. All rights reserved.";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Removing the hardcoded value override in pbxproj. From now on it will use the one set in `base.local.xcconfig`:
```
// To fix Xcode 15 warning: 'The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is
// set to 11.0, but the range of supported deployment target versions is 12.0 to 17.0.99.'
IPHONEOS_DEPLOYMENT_TARGET=12.0
```
☝️ add this to `Base.local.xcconfig` or run `make dependencies` to fix compiler warnings in Xcode 15

Reply:
With `Base.local.xcconfig` being uniformly set among all targets, this also fixes `IntegrationTests` build error in Xcode 15:
```
'Compiling for iOS 11.0, but module 'SRFixtures' has a minimum deployment target of iOS 12.0'
```